### DISCUSSION TITLE
Laggy

### DISCUSSION THREAD

**Discussion by bauhuynhdev** [rustdesk/rustdesk#5174] [General]
### Bug Description

I have set up Relay Server as my country's VPS IP.
But the lag is still happening.
My VPS has Firewall turned off.
I am using Rustdesk version 1.2.1

### How to Reproduce
This is when I use ping
`64 bytes from xxx: icmp_seq=1 ttl=49 time=14.2 ms`
`64 bytes from xxx: icmp_seq=2 ttl=49 time=14.0 ms`
`64 bytes from xxx: icmp_seq=3 ttl=49 time=14.0 ms`

### Expected Behavior

I have set up Relay Server as my country's VPS IP.
But the lag is still happening.

### Operating system(s) on local side and remote side

Windows 11 -> Windows 11, Ubuntu -> Windows 11

### RustDesk Version(s) on local side and remote side

1.2.1

### Screenshots

https://i.imgur.com/0d4LHxn.png

**Comment by dinger1986** [comment]
Did you only change your relay server?

**Comment by bauhuynhdev** [comment]
@dinger1986 
Yes sir.
https://i.imgur.com/30Wq7PL.png

**Comment by bauhuynhdev** [comment]
@dinger1986 
Do you mean I'm missing the Relay Server entry?

**Reply by dinger1986** [reply]
yeah maybe?

**Reply by bauhuynhdev** [reply]
I added and am checking again
